README.md

```diff
@@ -1,75 +1,26 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
+# Portafolio: Mobil Version
+This is a repo dedicated to share with you my experience
 
-Additional description about the project and its features.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
## [OPTIONAL]
- [ ] I think it will be nice if you put a screenshot of your first section in the `README` to make more professional

Reply:
## HI @andersonlebon !
Thanks for your feedback. I will put that in the future because I need to work on them first.

I'll merge it 👍